components/encryption/src/manager/mod.rs

```diff
@@ -473,6 +473,10 @@ impl DataKeyManager {
             // Encryption is being enabled.
             (Ok(None), _) => {
                 info!("encryption is being enabled. method = {:?}", args.method);
+                // If `master_key` doesn't work correctly, dict files will not be created.
+                let plaintext_temp = "";
```

Comment:
It would be better to have some content here. It would be valid to reject an encrypt operation because the content is empty.